src/tests/Common/wasm-test-runner/index.html

```diff
@@ -0,0 +1 @@
+<html><body><script type='module' src='test-main.js'></script></body></html>
```

Comment:
Should this be located next to test-main.js

Reply:
I could do it, but I don't like it to be named "index.html" when we place it next to test-main.js. And there isn't a simple to rename the file when copied to runtime tests.

@radical Do you know if we can set Xharness arg here (to open test-index.html instead of index.html. I know that xharness has an argument for that, but I'm not sure if we can it from `WasmTestRunner`)

Reply:
Keep it here. It ensures that dependencies are clean, so changes to `index.html` here will trigger wasm runtime tests.